portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,9 +31,6 @@
   @Override
   public void init() {
     comments = new ArrayList<String>();
```

Comment:
Alternatively, if you only need to instantiate comments object, but not populate it you can do so in line 29 where you declare it:
private ArrayList<String> comments = new ArrayList<>();

Either way is fine.

Reply:
Did not think of that, thank you!